README.md

```diff
@@ -239,6 +240,11 @@ and user may cancel it in process.
 * `iproto.Feature` type used instead of `ProtocolFeature`.
 * `iproto.IPROTO_FEATURE_` constants used instead of local ones.
 
+#### CRUD options
+
+* `crud` operations `Timeout` option has `crud.OptFloat64` type
+  instead of `crud.OptUint`.
+
```

Comment:
Please, create a separate heading for `crud package` as for other packages (pool, datetime etc).

Reply:
Updated